src/lib/Theme.ts

```diff
@@ -1016,6 +1042,23 @@ export default class Theme {
                 allowedDefaultProps,
             );
 
+            if(theme.is_private && hasAnyMetadataField){
```

Comment:
If theme is getting synced for the first time then `is_private` will be true @pooransuthar1999 ?

Reply:
Yes every theme created from cli will be a private theme only  so there key in the db will be is_private true